src/mesh/difops.cxx

```diff
@@ -695,45 +740,50 @@ const Field3D bracket(const Field3D &f, const Field2D &g, BRACKET_METHOD method,
         result(jx, jy, ncz - 1) = (Jpp + Jpx) * spacingFactor;
       }
     }
+#else
+    throw BoutException("BRACKET_ARAKAWA not valid with 3D metrics yet.");
```

Comment:
What makes `BRACKET_ARAKAWA` not work with 3d metrics? The `bracket(Field3D,Field3D)` version apparently does work...

Reply:
As far as I can tell, it's just `spacingFactor` -- but as you say, this was changed for the 3D-3D version, so should be possible here too? Am I missing something else?

Reply:
I think it should be "not implemented" - rather then "not valid".